src/fontra/views/editor/panel-reference-font.js

```diff
@@ -264,15 +292,114 @@ export default class ReferenceFontPanel extends Panel {
 
     this.controller.addKeyListener("charOverride", (event) => {
       this.editorController.canvasController.requestUpdate();
+      this.requestReferenceFontsPreview();
     });
 
     this.controller.addKeyListener("languageCode", (event) => {
       this.editorController.canvasController.setLangAttribute(this.model.languageCode);
+      this.requestReferenceFontsPreview();
     });
 
     this.editorController.canvasController.setLangAttribute(this.model.languageCode);
 
     referenceFontModel = this.model;
+
+    this.editorController.sceneSettingsController.addKeyListener(
+      ["selectedGlyph", "glyphLines"],
+      (event) => {
+        this.requestReferenceFontsPreview();
+      }
+    );
+  }
+
+  async requestReferenceFontsPreview() {
+    if (this.referenceFontsPreviewPromise) {
+      return this.referenceFontsPreviewPromise;
+    }
+    this.referenceFontsPreviewPromise = this.displayCurrentGlyphInReferenceFonts();
+    await this.referenceFontsPreviewPromise;
+    delete this.referenceFontsPreviewPromise;
+  }
+
+  async displayCurrentGlyphInReferenceFonts() {
+    const container = this.contentElement.querySelector(".reference-font-preview");
+    while (container.firstChild) {
+      container.removeChild(container.firstChild);
+    }
+    const selectedGlyphInfo = getSelectedGlyphInfo(
+      this.editorController.sceneSettings.selectedGlyph,
+      this.editorController.sceneSettings.glyphLines
+    );
+
+    let textToDisplay;
+
+    if (this.model.charOverride) {
+      textToDisplay = this.model.charOverride.charAt(0);
+    } else {
+      if (selectedGlyphInfo) {
+        if (selectedGlyphInfo.glyphName.includes(".")) {
+          const baseGlyphName = selectedGlyphInfo.glyphName.split(".")[0];
+          const codePoint = (this.editorController.fontController.glyphMap[
+            baseGlyphName
+          ] || [])[0];
+          if (codePoint) {
+            textToDisplay = String.fromCodePoint(codePoint);
+          }
+        } else {
+          textToDisplay = selectedGlyphInfo.character;
+        }
+      }
+    }
+
+    if (!textToDisplay) {
+      return;
+    }
+
+    if (
+      !textToDisplay &&
+      selectedGlyphInfo &&
+      selectedGlyphInfo.glyphName.includes(".")
+    ) {
+      const baseGlyphName = positionedGlyph.glyphName.split(".")[0];
+      const codePoint = (this.editorController.fontController.glyphMap[baseGlyphName] ||
+        [])[0];
+      if (codePoint) {
+        textToDisplay = String.fromCodePoint(codePoint);
+      }
+    }
+
+    if (!textToDisplay) {
+      return;
+    }
+
+    const currentCharacter = div(
+      {
+        class: "current-character",
+        style: `font-size: ${this.model.fontSize}px`,
+        lang: this.model.languageCode,
+      },
+      []
+    );
+    const rangeSlider = createDomElement("range-slider", {
+      type: "range",
+      value: this.model.fontSize,
+      minValue: 10,
+      maxValue: 200,
```

Comment:
Please make the max 300, make the default 100 (also elswhere in the code, maybe it should be a constant if it is currently hardcoded in several places), make sure the slider has a matching `defaultValue` property (it currently goes to 0 when you alt-click the slider).

Reply:
Elswhere, the default fo the font size is set to 14. Please make this 100, and use a constant for it.